components/text/ggrep/Makefile

```diff
@@ -62,22 +60,15 @@ unexport SHELLOPTS
 
 # Test transforms retain only relevant results output.
 COMPONENT_TEST_TRANSFORMS += \
-        '-n ' \
-        '-e "/TOTAL/p" ' \
-        '-e "/SKIP/p" ' \
-        '-e "/PASS/p" ' \
-        '-e "/FAIL/p" ' \
-        '-e "/ERROR/p" '
-
-# common targets
-build:		$(BUILD_64)
-
-install:	$(INSTALL_64)
-
-# 1 test fails - multibyte-white-space (something with illumos locales?)
-# 
-test:		$(TEST_64)
+	'-n ' \
+	'-e "/TOTAL/p" ' \
+	'-e "/SKIP/p" ' \
+	'-e "/PASS/p" ' \
+	'-e "/FAIL/p" ' \
+	'-e "/ERROR/p" '
 
+# Since version 3.4, GNU grep is needed for build.
+REQUIRED_PACKAGES += text/gnu-grep
```

Comment:
Very unfortunate... I think about ABI incompatibilities in ggrep dependencies...

Reply:
Can we somehow check that? I find it hard to imagine what might break, it's just a binary...?

Reply:
I think about some ABI-incompatible changes in libsigsegv or pcre. Anyway, it's not a reason for blocking PR.